src/components/Person/Person.jsx

```diff
@@ -1 +1,40 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+import './Person.scss';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  const partner = sex === 'f'
+    ? 'husband'
+    : 'wife';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+
+      {
+        age && (
+          <p className="Person__age">
+            {`I am ${age}`}
+          </p>
+        )
+      }
+
+      <p className="Person__partner">
+        {
+          isMarried
+            ? `${partnerName} is my ${partner}`
+            : 'I am not married'
```

Comment:
move ternary outside of jsx and save it to a variable, you don't want to have a lot of logic in jsx